build/infra_build.mk

```diff
@@ -169,3 +169,16 @@ dist-centos-5:
 		echo "[dist]   Creating executable: `basename $$main_package` in $(TARGET_DIR_CENTOS5)/`basename $$main_package`";\
 		$(GO_BIN_1_9) build -ldflags '$(LDFLAGS)' -o $(TARGET_DIR_CENTOS5)/`basename $$main_package` $$main_package || exit 1 ;\
 	done
+
+.PHONY: linux/harvest-tests
+linux/harvest-tests: GOOS=linux
+linux/harvest-tests: GOARCH=amd64
+linux/harvest-tests: go-get
+	go test ./test/harvest -tags="harvest" -v -c -o ./harvest-bin && ./harvest-bin
```

Comment:
if we are using a runners provisioned with Golang, we could avoid creating the binary and just running the go test

I guess this is fine for now, but we'll need to revamp this when adding custom runners